webapp/components/Dialog.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import MuiDialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import Typography from '@material-ui/core/Typography';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import { fontColor } from '../theme'
+import Paper from '@material-ui/core/Paper'
+import Box from '@material-ui/core/Box'
+import Link from '@material-ui/core/Link'
+import logo_nwb_explorer from '../resources/logos/nwb-explorer.png';
+import { NWB_WEBSITE } from './constants';
+
+const AboutContent = () => (
+  <Paper style={{ backgroundColor: "#4a4a4a", textAlign: 'center', padding: '15px 5px 5px' }}>
+    <img style={{ width: 150 }} src={logo_nwb_explorer} />
+    <Box m={1}>
+      <Typography variant="h5" style={{ color: fontColor }}>NWB Explorer v0.5.0</Typography>
+    </Box>
+    
+    <Box m={1} >
+      <Typography variant="body2" style={{ color: fontColor }}>
+        NWB Explorer is a web application that can be used by scientists to read, visualize and explore the content of NWB:N 2 files.
+      </Typography>
+    </Box>
+
+    <Box m={1} pb={2}>
+      <Typography variant="body2" style={{ color: fontColor }}>
+        Want to know more? Go to our <Link href={NWB_WEBSITE} target="_blank">website</Link>.
+      </Typography>
+    </Box>
+
+    <Box m={1}>
+      <Typography variant="body2" style={{ color: fontColor }}>
+        NWB Explorer is being developed in collaboration with:
+      </Typography>
+      <Link href="http://www.metacell.us" target="_blank">
+        <img style={{ width: 150, padding: "10px" }} src="https://raw.githubusercontent.com/ddelpiano/bucket/master/MetaCellLogoHorizontal300ppi.png"></img>
```

Comment:
This is coming from netpyne and should be improved also there. The image shouldn't come from outside and the css as well can be improved. "10px padding" is a kind of misuse we are trying to remove from the codebases as we can, unless there is a specific reason to keep things like that.